atlascope/core/migrations/0009_alter_dataset_dataset_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.12 on 2022-03-03 14:46
```

Comment:
I think this will end up conflicting with my remove-auth branch (that one also has a 0009 migration). A rebase and cleanup should help to fix the problem.

Reply:
should be fixed